__e2e__/verifySideNav.e2e.js

```diff
@@ -30,7 +30,7 @@ test.after(async () => {
   }
 })
 
-test.serial('should login successfully', async t => {
+test.serial('should login successfully', async e => {
```

Comment:
I prefer that this variable be kept `t` as it is the ava the test instance (not an error)